### DISCUSSION TITLE
Does toit support any kind of mesh network?

### DISCUSSION THREAD

**Discussion by bbinet** [toitlang/toit#2425] [General]
I'd like to provide network access to my esp32 devices through a mesh (wifi/ble/zigbee mesh) as some are located too far from the wifi access point.
Does Toit platform supports this use case?

**Comment by floitsch** [comment]
There isn't any specific library is package for this (yet), but it shouldn't be hard to use BLE or espnow (or any other RF) for this.
You would need to share the data explicitly, though.
